### PR TITLE
Added Option for Custom In Progress Messagee

### PR DESCRIPTION
**Pull request by mdewey**
## Description

Add the ability to customize the in-progress message for a form. Addresses some [accessiblity feedback](https://github.com/department-of-veterans-affairs/va.gov-team/issues/24641)

## Testing done (none yet) 
- [ ] unit 
- [ ] e2e

## Screenshots
screencapture-localhost-3001-health-care-health-questionnaires-questionnaires-answer-questions-introduction-2021-05-20-13_26_51

## Acceptance criteria
- [ ] "in-progress" message is an h2

## References
- [Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/24641)

### CODE REVIEW

File: src/platform/forms/save-in-progress/SaveInProgressIntro.jsx [hunk 1]
```diff
@@ -60,38 +60,52 @@ class SaveInProgressIntro extends React.Component {
 
         if (!isExpired) {
           const lastSavedDateTime = savedAt.format('M/D/YYYY [at] h:mm a');
-
-          alert = (
-            <div>
-              <div className="usa-alert usa-alert-info background-color-only schemaform-sip-alert">
-                <div className="schemaform-sip-alert-title">
-                  <strong>Your {appType} is in progress</strong>
```
Comment (bkjohnson) [inline src/platform/forms/save-in-progress/SaveInProgressIntro.jsx:68]: My understanding of the problem is that this `<strong>` would be better off as an `h3` in some situations and an `h2` in others. Is that correct?

**Review by bkjohnson — COMMENTED** [review]
I want to make sure I understand the problem and goal before I get too carried away :sweat_smile:

File: src/platform/forms/save-in-progress/SaveInProgressIntro.jsx (hunk 1 above)
Comment (mdewey) [reply src/platform/forms/save-in-progress/SaveInProgressIntro.jsx:68]: You are correct.

Comment (bkjohnson) [reply src/platform/forms/save-in-progress/SaveInProgressIntro.jsx:68]: I left some [feedback on the other PR](https://github.com/department-of-veterans-affairs/vets-website/pull/17282#discussion_r636359877) linked in the slack thread that I think is applicable here as well. That may result in fewer changes and a solution that is more general purpose. What do you think?

Comment (mdewey) [reply src/platform/forms/save-in-progress/SaveInProgressIntro.jsx:68]: Yes, I am all for using the other PR as the foundation for this fix as well. I like the Functional component solution you proposed I wasn't aware you could do stuff like that in React. it's pretty nifty. 

I am waiting to go further with this until that PR gets resolved.

**Comment by mdewey** [comment]
Moving to new PR.

**Pull request closed without merging**